tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -467,6 +467,12 @@ generic_config_updater/test_eth_interface.py::test_update_valid_invalid_index[33
       - "asic_type in ['mellanox']"
       - https://github.com/sonic-net/sonic-mgmt/issues/7733
 
+generic_config_updater/test_eth_interface.py::test_toggle_pfc_asym:
```

Comment:
The condition marker file should be sorted alphabetically

Reply:
Thanks for pointing out. Taken care of sorting the file alphabetically